pyproject.toml

```diff
@@ -31,24 +36,32 @@ classifiers = [
     "Topic :: Utilities",
 ]
 dependencies = [
-    "platformdirs>=2.2.0",
-    "typing_extensions>=4.0; python_version<'3.11'",
+    "platformdirs>=2.2",
+    "typing-extensions>=4; python_version<'3.11'",
 ]
 
 [project.optional-dependencies]
 numpy = [
-    "numpy>=1.6.0",
+    "numpy>=1.6",
 ]
-
 test = [
     "mypy",
     "pytest",
     "ruff",
 ]
 
 [project.urls]
-Homepage = "https://github.com/inducer/pytools/"
 Documentation = "https://documen.tician.de/pytools/"
+Homepage = "https://github.com/inducer/pytools/"
+
+[tool.setuptools.packages.find]
+include = [
+    "pytools*",
+]
```

Comment:
I also had to add this because otherwise the wheel would contain some other folders as well (e.g. doc).

Reply:
Hm, or not, I recall that being an issue with `modepy` or something.. Oh well..